src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureColor {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(int firstSide, int secondSide, String color) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + String.format("%.1f", getArea())
+                + " sq. units, first side: " + firstSide + " units, secondSide: " + secondSide
+                + " units, color: " + getColor());
```

Comment:
The message printed inside 'draw()' method says 'Figure: rectangle', but it should be 'Figure: right triangle' to match the class name and the type of figure it represents.